.github/workflows/dependencies.yml

```diff
@@ -1,27 +1,29 @@
-name: Bumper
+name: Update this project's version where it servers as a dependency
 on:
   push:
     branches:
       - master
   workflow_dispatch:
 
 jobs:
-  bumpProjects:
+  update_version:
+    # Pushes new refs to interested external repositorysitories, so they can do early testing against them.
     runs-on: ubuntu-latest
+    name: Update libp2p's version for ${{ matrix.target.repository }} from ${{ matrix.target.ref }}
     strategy:
       fail-fast: false
       matrix:
         target: [
-          { repo: status-im/nimbus-eth2, branch: unstable },
-          { repo: waku-org/nwaku, branch: master },
-          { repo: codex-storage/nim-codex, branch: master }
+          { repository: status-im/nimbus-eth2, ref: unstable },
```

Comment:
why does it need to be renamed from branch to ref?

Reply:
I'd argue it's a better choice since it can be any kind of ref, but most importantly it matches the `ref` parameter in `checkout` action, which is where it's used.